prophy/tests/test_class_helpers.py

```diff
@@ -0,0 +1,92 @@
+import prophy
+
+
+class SubStruct(prophy.with_metaclass(prophy.struct_generator, prophy.struct)):
```

Comment:
This makes tested code to execute before any test begins - not a good idea. Consider delaying creationg of types.